src/Microsoft.AspNetCore.SignalR.Core/Proxies.cs

```diff
@@ -17,13 +17,13 @@ public UserProxy(HubLifetimeManager<THub> lifetimeManager, string userId)
             _userId = userId;
         }
 
-        public Task SendAsync(string method, params object[] args)
```

Comment:
I noticed that when the reference was `IClientProxy` then a byte array would always be the first argument but with these typed proxies the `params` would take precedence. Just removed `params` on all the `IClientProxy` implementations.

Note that `client.SendAsync((object[])bytes);` will still expand out but that seems like a strange thing to do.